medusa/providers/generic_provider.py

```diff
@@ -460,7 +461,15 @@ def _get_episode_search_strings(self, episode, add_string=''):
                 episode_string += ('|', ' ')[len(self.proper_strings) > 1]
                 episode_string += episode.airdate.strftime('%b')
             elif episode.show.anime:
-                episode_string += '%02d' % int(episode.scene_absolute_number)
+                # If the showname is a season scene exception, we want to use the indexer episode number.
+                if (episode.scene_season > 1 and
+                    show_name in get_scene_exceptions(episode.show.indexerid, season=episode.scene_season,
+                                                      indexer=episode.show.indexer)):
+                    # This is apparently a season exception, let's use the scene_episode instead of absolute
+                    ep = int(episode.scene_episode)
+                else:
+                    ep = int(episode.scene_absolute_number)
+                episode_string += str(ep)
```

Comment:
Yeah I wasn't sure about that myself.. But as the default get_episode_string had it, I copied it.